### PR TITLE
Add Spell-Target-Position to "Summon Crypt Guard" spell

### PR DESCRIPTION
**Pull request by insunaa**
This PR adds the spell target position to the Summon Crypt Guard spell added here https://github.com/cmangos/mangos-wotlk/commit/4fa8d7e712ee3da45e955c35691609583e78112f

### CODE REVIEW

**Comment by AnonXS** [comment]
```SELECT * from `spell_template` where `id` = 29508;```

File: Updates/4325_naxxramas_spelllists.sql [hunk 1]
```diff
@@ -176,7 +179,160 @@ REPLACE INTO `creature_spell_targeting` (Id, Type, Param1, Param2, Param3, Comme
...
+(30119,0,181541,3),
+(30119,0,181542,3),
+(30119,0,181543,3),
+(30119,0,181544,3),
+(30119,0,181676,5),
+(30119,0,181677,3),
+(30119,0,181678,6);
+
 -- Various Fixes
+REPLACE INTO `scripted_areatrigger` VALUES
```
Comment (killerwife) [inline Updates/4325_naxxramas_spelllists.sql:330]: this goes to core

**Comment by killerwife** [comment]
Also spell lists for anubrekhan where :D

**Comment by insunaa** [comment]
https://github.com/cmangos/wotlk-db/blob/master/Updates/4325_naxxramas_spelllists.sql#L167 :>
Gr3zs was kind enough to prettify and merge it

File: Updates/4325_naxxramas_spelllists.sql (hunk 1 above)
Comment (insunaa) [reply Updates/4325_naxxramas_spelllists.sql:330]: it does indeed, let me remove it

**Pull request merged**